README.md

```diff
@@ -12,15 +12,27 @@ npm install dotmailer --save
 ```
 
 ## Usage
-```dotMailer(ApiFunction, tokens..., dataObject, callback)```
+```dotMailer({endpoint, tokens, config, cb})```
```

Comment:
```suggestion
```dotMailer({endpoint, tokens, config, callback})```
```
Sorry, being picky but cb may not be overly clear to some, should probably switch to promises at some point anyway

Reply:
yep happy to update..

Reply:
ha.. Promises & TS were coming in a PR in the next couple of wks

Reply:
done

Reply:
Promises 👍  Not fond of TypeScript though, would prefer to keep it JS; happy to accept a TS typings doc if possible though